docs/content/_navigation.json

```diff
@@ -994,9 +994,13 @@
             "title": "Utilities",
             "path": "/_apidocs/utilities"
           },
+          {
+            "title": "Pipes",
```

Comment:
should we call it "Dagster Pipes" or just "Pipes"?

Reply:
I think just "Pipes" to follow the convention of everything else in that menu

Reply:
ok 👌